dataclients/kubernetes/ingressdefinitions.go

```diff
@@ -172,3 +172,13 @@ type ClusterResourceList struct {
 	// Items, aka "resources".
 	Items []*ClusterResource `json:"resources"`
 }
+
+type secret struct {
+	Meta *definitions.Metadata `json:"Metadata"`
```

Comment:
nit: I think json field is lowercase and I would also name struct field exactly as a json field
```go
Metadata *definitions.Metadata `json:"metadata"`
```
but I guess it works either way